packages/client/src/components/FeaturedMovie.js

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+import {
+  Box,
+  CardMedia,
+  Card,
+  CardContent,
+  IconButton,
+  Typography,
+} from '@mui/material';
+import PlayArrowIcon from '@mui/icons-material/PlayArrow';
+
+export default function FeaturedMovie() {
+  return (
+    <Card>
+      <Box sx={{ position: 'relative', display: 'flex' }}>
+        <CardMedia
+          component="img"
+          sx={{
+            width: 1672,
+            height: 790,
+            borderRadius: 395,
+            zIndex: 1,
+            top: 238,
```

Comment:
It's better to follow some sort of a system for positions and spacing, so they look consistent to the user. A common practice is to only use multiples of 8.

Reply:
again because of the Figma all these numbers :D